### PR TITLE
#1510 Html nesting error in EtoMedia.tsx

### PR DESCRIPTION
**Pull request by AlexeyTuboltsev**
#1510 

turn \<p\> into \<div\>s.
add keys to MediaLinkEditor FormFields

### CODE REVIEW

File: app/components/eto/registration/pages/EtoMedia.tsx [hunk 1]
```diff
@@ -37,31 +37,33 @@ const EtoRegistrationMediaComponent = ({ savingData }: IProps) => (
     progressOptions={etoMediaProgressOptions}
   >
     <Section>
-      <p className="offset-1 mb-2 font-weight-bold">
+      <div className="offset-1 mb-2 font-weight-bold">
```
Comment (desfero) [inline app/components/eto/registration/pages/EtoMedia.tsx]: Do we need replace every `p` by `div`? P
Probably the issue is only with `Tooltip` component on line 52

Comment (trival) [reply app/components/eto/registration/pages/EtoMedia.tsx]: It throws a lot of console errors... if it holds more elements, it shouldn't be a 'p'

Comment (trival) [reply app/components/eto/registration/pages/EtoMedia.tsx]: but you are right, if it only contains a translation string, it should can stay a 'p', it is more semantic then

Comment (AlexeyTuboltsev) [reply app/components/eto/registration/pages/EtoMedia.tsx]: well, my thinking was, that these are not paragraphs in a text, but markup blocks, so the shouldn't be Ps. I don't insist though

Comment (desfero) [reply app/components/eto/registration/pages/EtoMedia.tsx]: Only 'div's on line 88 and 99 should be left as `p`. All other 'div's should be either 'div', 'label', 'legend' tags. But for now let's leave those `label` and `legends` as `div`s as fixing markup can take some time :( 
To summarise, let's revert back `div`s on line 88 and 99 to `p` and merge (and keep in mind that in future we need to fix markup :D)

is it ok? @trival @AlexeyTuboltsev everyone agrees?

File: app/components/shared/MediaLinksEditor.tsx [hunk 2]
```diff
@@ -45,12 +45,14 @@ const SingleMediaLinkField: React.SFC<
 
             return isFieldWithCharactersLimit ? (
               <FormField
+                key={`${props.name}.${fieldName}`}
```
Comment (desfero) [inline app/components/shared/MediaLinksEditor.tsx]: 👍

File: app/components/eto/registration/pages/EtoMedia.tsx (hunk 1 above)
Comment (AlexeyTuboltsev) [reply app/components/eto/registration/pages/EtoMedia.tsx]: yes, I'm totally ok with it. I'll correct it

**Pull request merged**